src/propulsion/calculate_RPM.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * Author: George Karabassis
+ * Organisation: HYPED
+ * Date: 30/3/2018
+ * Description:
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ *    except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *    either express or implied. See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "propulsion/calculate_RPM.hpp"
+
+#include <math.h>
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+#include <cstdint>
```

Comment:
only include the propulsion/calculare_RPM header file here. Move the rest to the calculate_RPM header file.

Reply:
Done it. Inly included the string and vector libraries, as lint requested it.